src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -128,5 +168,33 @@ public Player getPlayerAtPos(int row, int col){
         return board[row][col];
     }
 
+    public void resetBoard(){
+        initBoard();
+        winner = null;
+        currentPlayer = Player.X;
+        tieFlag = false;
+    }
+
+    public boolean isTie() {
+        return this.tieFlag;
+    }
 
+    private boolean checkTie() {
+        for (int i = 0; i < 3; i++)  {
+            for(int j = 0; j < 3; j++) {
+                if (board[i][j] == Player.NONE) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    private void incrementTies() {
```

Comment:
Unnecessary function. This is used only once and is a single line of code.

Reply:
Fixed in 82833d6